packages/maps/src/components/basic/GeoDistanceDropdown.js

```diff
@@ -589,12 +591,13 @@ GeoDistanceDropdown.propTypes = {
 };
 
 GeoDistanceDropdown.defaultProps = {
+	autoLocation: true,
 	className: null,
 	placeholder: 'Select a value',
 	showFilter: true,
 	style: {},
+	suggestions: [],
```

Comment:
This won't help because `suggestions: state.hits[props.componentId] && state.hits[props.componentId].hits` will overwrite suggestions value to `undefined` when hits are not available. It'd be best if you assign the defaultValue in the `mapStateToProps` function.

Reply:
Updated!